### ISSUE TITLE
Can't create private repo with MkGihub

### ISSUE THREAD

**Issue by g4s8** [jcabi/jcabi-github#1421]
`MkGithub.repos.create()` ignores `private` flag and always create public repository.
Here is test to reproduce:
```java
final boolean priv = true;
MatcherAssert.assertThat(
  new Repo.Smart(
    new MkGithub().repos().create(
      new Repos.RepoCreate("test", priv)
    )
  ).isPrivate(),
  Matchers.is(priv)
);
```

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C9R53K5JA) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by amihaiemil** [comment]
@g4s8 thanks, we'll look into it

**Comment by 0crat** [comment]
Job [#1421](https://github.com/jcabi/jcabi-github/issues/1421) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
The job [#1421](https://github.com/jcabi/jcabi-github/issues/1421) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C9R53K5JA/1631712607); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting on review/merge of PR #1423

**Comment by 0crat** [comment]
> @0crat waiting on review/merge of PR #1423  ([here](https://github.com/jcabi/jcabi-github/issues/1421#issuecomment-398401217))

@llorllale The impediment for [#1421](https://github.com/jcabi/jcabi-github/issues/1421) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by amihaiemil** [comment]
@0crat boost x4

**Comment by amihaiemil** [comment]
@0crat boost 4x

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@095c162d-96f5-4418-a96c-3a3627c066c0, thread: pool-1131-thread-1
java.util.concurrent.ThreadPoolExecutor$AbortPolicy[2063] java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@835c8b2 rejected from java.util.concurrent.ThreadPoolExecutor@751833d7[Shutting down, pool size = 8, active threads = 8, queued tasks = 21, completed tasks = 102725]
```

[0.22.6](https://github.com/zerocracy/farm): Issue: jcabi/jcabi-github#1421, Comment: 398410675

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@095c162d-96f5-4418-a96c-3a3627c066c0, thread: pool-1131-thread-1
com.zerocracy.radars.github.ReSafe[82] java.io.IOException: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@835c8b2 rejected from java.util.concurrent.ThreadPoolExecutor@751833d7[Shutting down, pool size = 8, active threads = 8, queued tasks = 21, completed tasks = 102725]
java.util.concurrent.ThreadPoolExecutor$AbortPolicy[2063] java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@835c8b2 rejected from java.util.concurrent.ThreadPoolExecutor@751833d7[Shutting down, pool size = 8, active threads = 8, queued tasks = 21, completed tasks = 102725]
```

[0.22.6](https://github.com/zerocracy/farm): Issue: jcabi/jcabi-github#1421, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@095c162d-96f5-4418-a96c-3a3627c066c0, thread: pool-1132-thread-1
java.util.concurrent.ThreadPoolExecutor$AbortPolicy[2063] java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@2c6849a4 rejected from java.util.concurrent.ThreadPoolExecutor@751833d7[Shutting down, pool size = 8, active threads = 8, queued tasks = 21, completed tasks = 102725]
```

[0.22.6](https://github.com/zerocracy/farm): Issue: jcabi/jcabi-github#1421, Comment: 398655016

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@095c162d-96f5-4418-a96c-3a3627c066c0, thread: pool-1132-thread-1
com.zerocracy.radars.github.ReSafe[82] java.io.IOException: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@2c6849a4 rejected from java.util.concurrent.ThreadPoolExecutor@751833d7[Shutting down, pool size = 8, active threads = 8, queued tasks = 21, completed tasks = 102725]
java.util.concurrent.ThreadPoolExecutor$AbortPolicy[2063] java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@2c6849a4 rejected from java.util.concurrent.ThreadPoolExecutor@751833d7[Shutting down, pool size = 8, active threads = 8, queued tasks = 21, completed tasks = 102725]
```

[0.22.6](https://github.com/zerocracy/farm): Issue: jcabi/jcabi-github#1421, Action: `created`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@095c162d-96f5-4418-a96c-3a3627c066c0, thread: pool-1135-thread-1
com.zerocracy.radars.github.ReSafe[82] java.io.IOException: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@48f2798a rejected from java.util.concurrent.ThreadPoolExecutor@7e91cbd3[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 2432938]
java.util.concurrent.ThreadPoolExecutor$AbortPolicy[2063] java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@48f2798a rejected from java.util.concurrent.ThreadPoolExecutor@7e91cbd3[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 2432938]
```

[0.22.6](https://github.com/zerocracy/farm): Issue: jcabi/jcabi-github#1421, Action: `created`

**Comment by 0crat** [comment]
> @0crat boost x4 ([here](https://github.com/jcabi/jcabi-github/issues/1421#issuecomment-398410675))

@amihaiemil Argument "x4" doesn't match regex `\d+x`

**Comment by amihaiemil** [comment]
@0crat boost 4x

**Comment by 0crat** [comment]
> @0crat boost 4x ([here](https://github.com/jcabi/jcabi-github/issues/1421#issuecomment-398657423))

@amihaiemil Boost 4x was set for [#1421](https://github.com/jcabi/jcabi-github/issues/1421)

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
Boosting tasks is against our principles, see [§15](http://www.zerocracy.com/policy.html#15): -10 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/jcabi/jcabi-github/issues/1421#issuecomment-398657882))

@g4s8 This is what I know about this job in [C9R53K5JA](https://www.0crat.com/p/C9R53K5JA), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1421](https://github.com/jcabi/jcabi-github/issues/1421) is [in scope](http://www.zerocracy.com/policy.html#14) for 8days
  * The role is `DEV`
  * The job is assigned to @llorllale[/z](https://www.0crat.com/u/llorllale) for [8days](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job has an [impediment](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 60 minutes/points
  * These users are banned and won't be assigned:
    * @g4s8[/z](https://www.0crat.com/u/g4s8): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C9R53K5JA?q=%7Bjob%3A%27gh%3Ajcabi%2Fjcabi-github%231421%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by amihaiemil** [comment]
@rultor release it, tag is `0.40`

**Comment by amihaiemil** [comment]
@rultor try to release again, tag is `0.40`

**Comment by llorllale** [comment]
@amihaiemil relevant part of error:

> [ERROR] * Artifact updating: Repository ='releases:Releases' does not allow updating artifact='/com/jcabi/jcabi-github/0.40/jcabi-github-0.40.jar'

**Comment by amihaiemil** [comment]
@llorllale I know, but I have no idea what that means. Maybe the release to maven central has been performed on the first trial (there, it seems to have been a timeout issue), and now it won't allow overriding. let's try ``0.41``....

**Comment by amihaiemil** [comment]
@rultor release, tag is `0.41`

**Comment by amihaiemil** [comment]
@g4s8 Version ``0.41``, containing the fix, is now released. Can you close this?

**Comment by g4s8** [comment]
@amihaiemil yes, thanks

**Comment by 0crat** [comment]
Order was finished: +60 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#1421](https://github.com/jcabi/jcabi-github/issues/1421) is now out of scope
